pkg/autoupdate/autoupdate.go

```diff
@@ -44,15 +44,16 @@ type Controller struct {
 	client        clientset.Interface
 	eventRecorder record.EventRecorder
 
-	syncHandler func(key string) error
+	syncHandler       func(key string) error
+	statusSyncHandler func(key string) error
```

Comment:
not used.

Reply:
Yeah, I was debating removing these.  In practice we test successive iterations of calling sync, and testing that informers work hasn’t been a high priority and/or is better covered by e2e tests